tests/unit-tests/reports/overview/data-provider/test-class-sensei-reports-overview-data-provider-students.php

```diff
@@ -320,7 +336,7 @@ private function createUserWithCourseEnrollment( array $user_args = [], int $cou
 		$user_id   = $this->factory->user->create( $user_args );
 		$course_id = $course_id ?? $this->factory->course->create();
 
-		$this->directlyEnrolStudent( $user_id, $course_id );
+		$this->manuallyEnrolStudentInCourse( $user_id, $course_id );
```

Comment:
It can't be the previous approach because it didn't trigger the recalculation. And now we're trusting on that.